pandas/tests/test_frame.py

```diff
@@ -12053,6 +12054,60 @@ def test_quantile(self):
         expected = Series([3., 4.], index=[0, 1])
         assert_series_equal(result, expected)
 
+    def test_quantile_interpolation(self):
+        from numpy import percentile
+        #interpolation = linear (default case)
+        q = self.tsframe.quantile(0.1, axis=0,interpolation='linear')
+        self.assertEqual(q['A'], percentile(self.tsframe['A'], 10))
```

Comment:
you can also use `np.percentile` here, instead of importing percentile explicitely from numpy (np is already available)